delatore/sources/influx.py

```diff
@@ -296,21 +297,16 @@ async def _get_status(self, metric):
         """Generate status, line per server"""
         query = metric.query.format(entity=metric.metric_id)
         last_record = await self.influx_client.query(query)
-        last_time_ms = None
         if not self._host_timings:
             for series in last_record.raw['series']:
-                host = series['tags']['server']
-                self._host_timings[host] = deque([], self.DEQUE_SIZE)
-
+                hostname = series['tags']['server']
+                self._host_timings[hostname] = deque([], self.DEQUE_SIZE)
         statuses = {}
         for series in last_record.raw['series']:
             try:
                 hostname = series['tags']['server']
-            except KeyError:
-                continue
-            try:
-                query_time = series['values'][0][0]
-                last_time_ms = _convert_time(query_time)
+                last_time = series['values'][0][0]
```

Comment:
`last_time, last_timems, * = series[values][0]` can be put in try... block alone

Reply:
we will obtain less readable code

Reply:
then just move `series['values'][0]` to vaiable